packages/tasks/src/tasks/mask-generation/about.md

```diff
@@ -0,0 +1,7 @@
+## About the Task
+
+Mask Generation task involves generating masks/segments for objects in a given image. It could be zero shot or based on training data.
```

Comment:
```suggestion
Mask generation task involves generating masks for meaningful segments in a given image. It could be zero-shot or based on training data.
```